test/models/mainstream_browse_page_test.rb

```diff
@@ -26,6 +26,18 @@
     it "returns the browse page description" do
       assert_equal "Information about eligibility, claiming and when Child Benefit stops", @page.description
     end
+
+    describe "#second_level_pages_curated?" do
+      it "returns false when second_level_ordering = alphabetical" do
+        @api_data["details"]["second_level_ordering"] = "alphabetical"
+        assert_equal false, @page.second_level_pages_curated?
+      end
+
+      it "returns true when second_level_ordering != curated" do
+        @api_data["details"]["second_level_ordering"] = "curated"
+        assert_equal true, @page.second_level_pages_curated?
+      end
+    end
```

Comment:
These unit tests could do with being extended to cover the cases where the details hash is missing, or the second_level_ordering is unspecified.